lib/bolt/executor.rb

```diff
@@ -64,15 +66,18 @@ def on(nodes, callback = nil)
 
       results_to_hash(results)
     end
+    private :on
 
     def summary(action, object, result)
       fc = result.select { |_, r| r.error }.length
       npl = result.length == 1 ? '' : 's'
       fpl = fc == 1 ? '' : 's'
       "Ran #{action} '#{object}' on #{result.length} node#{npl} with #{fc} failure#{fpl}"
     end
+    private :summary
 
-    def run_command(nodes, command)
+    def run_command(targets, command)
+      nodes = from_targets(targets)
```

Comment:
Is there a reason to call this in the run_ methods instead of in on?

Reply:
I felt it made the call to `on` slightly more readable. The alternative would pass targets, but then still get a `node` passed to the block.